packages/base/src/delegate/ItemNavigation.ts

```diff
@@ -102,9 +102,7 @@ class ItemNavigation {
 
 		this.rootWebComponent = rootWebComponent;
 		this.rootWebComponent.addEventListener("keydown", this._onkeydown.bind(this));
-		this.rootWebComponent._onComponentStateFinalized = () => {
-			this._init();
-		};
+		this.rootWebComponent._addItemNavigation(this);
```

Comment:
Currently `UI5Element.ts` does not have a notion about Item Navigation at all. This change makes them more tightly coupled. 

I understand the issue and will think about a solution and suggest a simpler fix in this, or another change